gpt_json/gpt.py

```diff
@@ -91,6 +91,7 @@ class RunResponse(Generic[SchemaType], BaseModel):
     fix_transforms: FixTransforms | None
     function_call: Callable[[BaseModel], Any] | None
     function_arg: BaseModel | None
+    usage: dict[str, int] | None = None
```

Comment:
Assuming this usage dict is documented in the OpenAI API and unlikely to break compatibility over time, let's refactor into a pydantic model so we get typehints about the expected values?